pgd_search/tests.py

```diff
@@ -857,6 +858,49 @@ def test_saving_image(self):
         response = self.driver.find_element_by_id("button-save").click()
 
 class ViewTest(TestCase):
-    def home_page_noerror(self):
+
+
+    #Method must start with "test"
+    def test_home_page_noerror(self):
         response = self.client.get(reverse('/'))
         self.assertEqual(response.status_code, 200)
+
+
+
+class CheckDumpTest(TestCase) :
+
+
+    fixtures = ['pgd_search.json']
+
+    def test_download_tsv(self):
+
+        search = Search(segmentLength=5)
+
+        # create associated Search_residues
+        data = {}
+        #data['index'] = 5
+        data['residues'] = 5
+        data['occm_-1'] = 0.58
```

Comment:
Altering the corresponding values will ensure that you've got them in the right order.

I would consider splitting the lines on tab characters, catching the headers and the output.  You could then confirm that the headers are in the correct order, and that the data is in the correct columns.